.github/workflows/gradle-check.yml

```diff
@@ -16,24 +16,8 @@ permissions:
   contents: read # to fetch code (actions/checkout)
 
 jobs:
-  check-files:
```

Comment:
I haven't found the replacement at the moment, so just removing the compromised action (consequence - we'll run full checks all the time).